packages/vscode-extension/package.nls.json

```diff
@@ -410,5 +433,17 @@
     "teamstoolkit.walkthroughs.steps.teamsToolkitExploreMore.title": "What's next?",
     "teamstoolkit.walkthroughs.steps.teamsToolkitPreview.description": "Press [F5](command:fx-extension.selectAndDebug?%5B%22WalkThrough%22%5D) or discover '[Run and Debug](command:workbench.view.debug)' panel on the activity bar, and click the play icon to locally preview your app in Teams context.\n[Run Local Preview (F5)](command:fx-extension.selectAndDebug?%5B%22WalkThrough%22%5D)\n__Tip: To run local preview, sign in to Microsoft 365 (organizational account) with custom app upload option.__",
     "teamstoolkit.walkthroughs.steps.teamsToolkitPreview.title": "Preview your Teams app locally",
-    "teamstoolkit.walkthroughs.title": "Get Started with Teams Toolkit"
+    "teamstoolkit.walkthroughs.title": "Get Started with Teams Toolkit",
+    "teamstoolkit.officeAddIn.terminal.start": "start office dev add in terminal...",
+    "teamstoolkit.officeAddIn.terminal.open": "Run tasks in this terminal for office add in",
+    "teamstoolkit.officeAddIn.terminal.installDependency": "Installing dependency...",
+    "teamstoolkit.officeAddIn.terminal.validateManifest": "Validating manifest...",
+    "teamstoolkit.officeAddIn.terminal.startDebugging":"Starting debugging...",
+    "teamstoolkit.officeAddIn.terminal.stopDebugging": "Stopping debugging...",
+    "teamstoolkit.officeAddIn.terminal.generateManifestGUID": "Generating manifest GUID...",
+    "teamstoolkit.officeAddIn.terminal.terminate": "* Ctrl +C to close the terminal.",
+    "teamstoolkit.officeAddIn.terminal.fail.tips": "failed and check the error message!",
+    "teamstoolkit.officeAddIn.terminal.success.tips": "completed successfully!",
+    "teamstoolkit.officeAddIn.terminal.manifest.notfound":"cannot find the manifest xml file"
```

Comment:
Change this to - Manifest xml file not found
Logic: To maintain the consistency with other strings.